packages/wdio-cli/src/utils.ts

```diff
@@ -840,177 +835,11 @@ export function detectPackageManager(argv = process.argv) {
     )) || 'npm'
 }
 
-/**
- * add ts-node if TypeScript is desired but not installed
- */
-export async function setupTypeScript(parsedAnswers: ParsedAnswers) {
```

Comment:
Just to verify, if a user creates a new WDIO project and decides to use TypeScript for it, a `tsconfig.json` won't be needed then? This section would generate such a file with good defaults based on the setup a user has selected.

Reply:
Good catch, I'll reinstate this function